.github/CODEOWNERS

```diff
@@ -51,17 +51,18 @@
 /examples/demo_search/ @elastic/kibana-app-services
 /examples/developer_examples/ @elastic/kibana-app-services
 /examples/embeddable_examples/ @elastic/kibana-app-services
-/examples/embeddable_explorer/ @elastic/kibana-app-services
```

Comment:
is `/examples/embeddable_explorer/` gone?  In all "current" branches?  If not, we should leave it.

Reply:
Oh, snap! my mistake thought this is a duplicate of `/examples/embeddable_examples/` and removed it. Brought it back now